README.md

```diff
@@ -4,8 +4,11 @@
 [![Docker Pulls](https://img.shields.io/docker/pulls/haugene/transmission-openvpn.svg)](https://hub.docker.com/r/haugene/transmission-openvpn/)
 [![Join the chat at https://gitter.im/docker-transmission-openvpn/Lobby](https://badges.gitter.im/docker-transmission-openvpn/Lobby.svg)](https://gitter.im/docker-transmission-openvpn/Lobby?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 
+
 ## Quick Start
 
+## WARNING : this fork is only a working fork... always refer to the orginal Haugene repository
```

Comment:
This should be removed if it is to be merged

Reply:
Sorry... i  am not very aware a about merging an excluding files.. i will do that asap. TY